.github/workflows/pr-checks.yaml

```diff
@@ -90,44 +85,57 @@ jobs:
 
-          curl -Lo opm https://github.com/operator-framework/operator-registry/releases/download/v${OPM_VERSION}/linux-amd64-opm
-          chmod +x opm
-          mv -v opm $GITHUB_WORKSPACE/bin/
 
-          curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v${K8S_VERSION}/bin/linux/amd64/kubectl
-          chmod +x kubectl
-          mv -v kubectl $GITHUB_WORKSPACE/bin/
+  acceptance_non_olm:
+    name: Non OLM Acceptance Tests with Kubernetes
+    runs-on: ubuntu-20.04
 
-          curl -Lo minikube https://storage.googleapis.com/minikube/releases/v${MINIKUBE_VERSION}/minikube-linux-amd64
-          chmod +x minikube
-          mv -v minikube $GITHUB_WORKSPACE/bin/
+    env:
+      EXTRA_BEHAVE_ARGS: "--tags=~@knative --tags=~@openshift --tags=~@olm"
+      UMOCI_VERSION: "0.4.5"
+      OPERATOR_VERSION: "0.4.0"
 
-      - name: Set KUBECONFIG
-        run: |
-          KUBECONFIG=$GITHUB_WORKSPACE/miniconfig
-          echo "KUBECONFIG=$KUBECONFIG" >> $GITHUB_ENV
-          touch $KUBECONFIG
+    steps:
+      - name: Checkout Git Repository
+        uses: actions/checkout@v2
 
-      - name: Set up minikube
-        run: |
-          ./hack/start-minikube.sh start --kubernetes-version=v${K8S_VERSION} --driver=$CONTAINER_RUNTIME --cpus $(nproc) --memory 5g
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "^1.15.6"
 
-      - name: Docker Info
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
+          architecture: "x64"
+
+      - name: Set up CLI
+        uses: ./.github/actions/setup-cli
+
+      - name: Setup umoci cli
         run: |
-          eval $(minikube docker-env)
-          docker ps
-          kubectl get nodes -o yaml
-          kubectl cluster-info
-          docker info
+          curl -Lo umoci https://github.com/opencontainers/umoci/releases/download/v${UMOCI_VERSION}/umoci.amd64
+          chmod +x umoci
+          mv -v umoci $GITHUB_WORKSPACE/bin/
 
-      - name: Acceptance tests
+      - name: Acceptance tests with release artifacts
         timeout-minutes: 60
         run: |
           eval $(minikube docker-env)
-          make OPERATOR_REPO_REF=$(minikube ip):5000/sbo SKIP_REGISTRY_LOGIN=true release-operator -o registry-login test-acceptance-with-bundle
\ No newline at end of file
+          GIT_COMMIT_ID=$(git rev-parse --short=8 HEAD)
+          make OPERATOR_REPO_REF=$(minikube ip):5000/sbo SKIP_REGISTRY_LOGIN=true release-operator -o registry-login
+          make TLS_VERIFY=false OPERATOR_REPO_REF=$(minikube ip):5000/sbo OPERATOR_BUNDLE_IMAGE_REF=$(minikube ip):5000/sbo:bundle-${OPERATOR_VERSION}-$GIT_COMMIT_ID SKIP_REGISTRY_LOGIN=true release-manifests -o registry-login
+          namesp=$(kubectl config view --minify --output 'jsonpath={..namespace}')
+          sed -i "s/REPLACE_NAMESPACE/$namesp/g" out/release.yaml
```

Comment:
The actual release.yaml would have REPLACE_NAMESPACE. For testing purposes we are just renaming in the CI and not in the actual one.

Reply:
We should not have it. Any reasons for that?

Reply:
Also we cant omit to have REPLACE_NAMESPACE because if we do we encounter this error
```
Error from server (Invalid): error when creating "out/release.yaml": ClusterRoleBinding.rbac.authorization.k8s.io "service-binding-operator" is invalid: subjects[0].namespace
```

if we have a subject we need a ns for it to be applied successfully

Reply:
The more I think about this, the more I would like to see `release.yaml` to be fully ready for user consumption, i.e. it should be possible to perform the following:

```
kubectl apply -f release.yaml
```

As the result, the operator will be installed in a predefined documented namespace. That should be default and would lower the barrier a lot. If somebody wants to drift from that, he already knows how to do it. The target namespace could be passed as arg to ` hack/release-manifest.py` script.

Therefore, let's create `release.yaml` file that creates all resources in `service-binding-operator` namespace, add the proper `Namespace` object to the file as well as the first entry.